### PR TITLE
Fixing issues

### PR DESCRIPTION
**Pull request by ggabernet**
Fixes issues:
#57 #50 #45 
- added functionality
- added github actions

## PR checklist

- [x] This comment contains a description of changes (with reason)
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If necessary, also make a PR on the [nf-core/bcellmagic branch on the nf-core/test-datasets repo]( https://github.com/nf-core/test-datasets/pull/new/nf-core/bcellmagic)
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [x] Make sure your code lints (`nf-core lint .`).
- [x] Documentation in `docs` is updated
- [x] `CHANGELOG.md` is updated
- [ ] `README.md` is updated

**Learn more about contributing:** https://github.com/nf-core/bcellmagic/tree/master/.github/CONTRIBUTING.md

### CODE REVIEW

File: nextflow.config [hunk 1]
```diff
@@ -15,13 +15,17 @@ params {
 
   // Initializing parameters
   set_cluster_threshold = false
+  cluster_threshold = -1
```
Comment (apeltzer) [inline nextflow.config]: There is no docs for that one ;-)

File: nextflow.config [hunk 2]
```diff
@@ -15,13 +15,17 @@ params {
...
   igblast_base = false
   imgtdb_base = false
   metadata = false
   saveDBs = false
   index_file = false
   cprimers = false
   vprimers = false
+  downstream_only = false
```
Comment (apeltzer) [inline nextflow.config]: Documentation ;-)

File: nextflow.config [hunk 3]
```diff
@@ -15,13 +15,17 @@ params {
...
+  umi_position = 'R1'
```
Comment (apeltzer) [inline nextflow.config]: Documentation ;-)

File: nextflow.config [hunk 4]
```diff
@@ -15,13 +15,17 @@ params {
...
+  skipDownstream = false
```
Comment (apeltzer) [inline nextflow.config]: Documentation ;-)

File: nextflow.config (hunk 1 above)
Comment (ggabernet) [reply nextflow.config]: just added them in the index :)

**Pull request merged**